src/submit.py

```diff
@@ -26,6 +26,33 @@ async def _git_push(repo_dir: pathlib.Path) -> tuple[int, str]:
     return return_code, output
 
 
+async def _create_submission_commit(repo_dir: pathlib.Path):
+    # Add empty commit
+    process = await asyncio.subprocess.create_subprocess_exec(
+        "git",
+        "commit",
+        "--allow-empty",
+        "-m", "SUBMISSION",
+        cwd=repo_dir,
+        stdout=asyncio.subprocess.PIPE,
+        stderr=asyncio.subprocess.STDOUT,
+    )
+    await process.communicate()
```

Comment:
This technically doesn't always create an empty commit. The repo could contain uncommitted changes, and i don't think we'd want to commit those. Should instead probably do something like `git stash && git commit --allow-empty && git stash pop`

Reply:
oh, right - it only asks if it should proceed with uncommited changes, it doesn't actually abort